tools/slack/guidelines.md

```diff
@@ -76,6 +76,31 @@ Slack Groups allow you to direct messages to a specific list of people in a more
 
 Groups are a great way to alert people who might not be in a channel about something that needs their attention or make sure urgent incidents are directed to the right people and not an entire channel. For example, you can ping **@github-admins** in #admins-github if you have an urgent issue instead of using an **@channel** or use **@federalist-team** to alert that group of a conversation in another channel that we should see immediately.
 
+## Custom emoji
+
+Emoji are an important part of our communication within Slack. You are welcome
+to add custom emoji. Keep in mind that interactions in Slack are covered by our
+[Code of Conduct]({% link _pages/policies/conduct-policies/code-of-conduct.md %}), including emoji.
+
+To create custom emoji:
+
+1. Open [#emoji-showcase](https://gsa-tts.slack.com/archives/C0X2T36AY) in Slack
```

Comment:
Does it have to be #emoji-showcase, or can I do this from anywhere?

Reply:
It does not _have_ to be, but it simplifies the Slack Workflow. Currently the Workflow publishes the approval to where the request was initiated and I kind of like keeping the request and approval notice in #emoji-showcase. It builds a sense of community there and it trains you that "any emoji I request is going to show up here like these other approvals did". It's almost like a transparency log for emoji creation.

Would you prefer something else?

Reply:
No, I really just wondered. It might be helpful to add some version of that in the doc... that you _can_ do it from anywhere, but if you keep it in #emoji-community it keeps things tidy and nice for the admins. Maybe?